core-services/prow/02_config/_boskos.yaml

```diff
@@ -994,6 +994,39 @@ resources:
   - ap-northeast-1--fleet-manager-qe-quota-slice-2
   state: free
   type: fleet-manager-qe-quota-slice
+- names:
```

Comment:
I also wonder whether we actually need this additional cluster-profile, since the books config for the non-arm64 dev one is using only the us-central1 zone... @deepsm007 wdyt?

Reply:
IMHO separation of concerns helps.

Reply:
Better to have separate cluster-profile, it allows us to control zone we want to use in future.

Reply:
is arm64 available in us-central?

Reply:
> are arm64 instances available in us-central?

Yes.

Please refer https://github.com/openshift/release/blob/ec17dd1983ff77374a8dcf7da9085b9a602bd7a3/core-services/prow/02_config/generate-boskos.py#L138-L146